interchain-token-service/src/primitives.rs

```diff
@@ -56,13 +64,13 @@ pub enum ItsHubMessage {
     /// ITS edge source contract -> ITS Hub
     SendToHub {
         /// True destination chain of the ITS message
-        destination_chain: ChainName,
+        destination_chain: ChainNameRaw,
```

Comment:
I'll change this back to `ChainName` since it'll auto normalize the underlying string so I don't have to normalize explicitly. A bit annoying that source has to use the raw name, but we can't get rid of legacy naming in ITS without breaking consistency.

Reply:
what do you mean by that? Why can't we use `ChainName` at the source?

Reply:
The `source_chain` comes is the case sensitive identifier that comes from the `message.cc_id.source_chain`. ITS allows optionally calling a contract along with the transfer (i.e GMP call with token). Similar to GMP, the app will receive the true `source_chain` and might have a whitelist based on the string. Hence, the raw chain name needs to be passed to avoid inconsistencies with GMP